packages/jupyter-ai/jupyter_ai/handlers.py

```diff
@@ -116,7 +116,7 @@ def get_chat_user(self) -> ChatUser:
         """Retrieves the current user. If `jupyter_collaboration` is not
         installed, one is synthesized from the server's current shell
         environment."""
-        collaborative = self.config.ServerApp.jpserver_extensions.get_value({}).get(
```

Comment:
I wouldn't recommend going through the config object, personally. 

Instead, I would check the server's `extension_manager` API for the "true" value. The ExtensionManager API could certainly be improved but it's the source of truth for loaded + linked server extensions.

It would look something like this:
```python
# Get a dictionary of all loaded extensions. (`serverapp` is a property on all `JupyterHandler` subclasses)
extensions = self.serverapp.extension_manager.extensions
collaborative = "jupyter_collaboration" in extensions and extensions["jupyter_collaboration"].enabled
```

Reply:
Thanks Zach! @abbott This change is simple, can you help make it?

Reply:
We're all set. Much appreciated @Zsailer and @dlqqq